pbr/scene/robot.py

```diff
@@ -62,7 +63,8 @@ def construct(self, robot_info):
                     col_path = os.path.join(tex_path, file)
                 if re.search(nor_re, file, re.I) is not None:
                     nor_path = os.path.join(tex_path, file)
-
+            print(f"col_path: {col_path}")
+            print(f"nor_path: {nor_path}")
```

Comment:
should these be here, looks like debugging?

Reply:
forgor :((